Knock.net/Resources/Tenants/Tenant.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Knock
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Contains information about a Knock Tenant record.
+    /// </summary>
+    public class Tenant
+    {
+        /// <summary>
+        /// The id of the tenant
+        /// </summary>
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        /// <summary>
+        /// The data associated
+        /// </summary>
+        [JsonProperty("data")]
```

Comment:
I don't think this is right given that we're returning this as `properties` from the API